.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--
+$ go fmt ./...
+
+At this point, code style is not guaranteed within Github Action and/or Circle CI.
+Please run `go fmt ./...` from the root directory.
+
+@see https://github.com/StackExchange/dnscontrol/pull/1996#issuecomment-1407388782
+
+$ go mod tidy
```

Comment:
We should probably ask anyone to run 3 commands:

---

Before you submit a pull request, please make sure you've run the following commands from the root directory.

```
go fmt ./...
go generate
go tidy
```

Reply:
That's also an option!